resources/locale/en.yml

```diff
@@ -89,6 +89,7 @@ fof-polls:
       subtitle_placeholder: Subtitle/Description (Optional)
       submit: Submit
       delete: Delete
+      error: Oops! Something went wrong. Please reload the page and try again.
```

Comment:
Rather than re-implementing this translation, let's link the key to core's implementation ;)

`error: => core.lib.error.generic_message`